int/api/massa/version.go

```diff
@@ -0,0 +1,19 @@
+package massa
+
+import (
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/station/api/swagger/server/models"
+	"github.com/massalabs/station/api/swagger/server/restapi/operations"
+	"github.com/massalabs/station/int/config"
+)
+
+func NewGetMassaStationVersion() operations.GetMassaStationVersionHandler {
+	return &getMassaStationVersion{}
+}
+
+type getMassaStationVersion struct{}
```

Comment:
let's try to avoid declaring a empty struct. I think we can return a getMassaStationVersionFunc:

```
operations.GetMassaStationVersionHandler = getMassaStationVersionFunc
```